.travis.yml

```diff
@@ -1,15 +1,14 @@
-sudo: true
-dist: trusty
+os: linux
 
-language: python
+language: python 
 
-os:
-    - linux
+python:
+    - "2.7"
+    - "3.5"
```

Comment:
Should we also add a run on Python 3.8 just to make sure things work OK there?

What is the timeframe for dropping support for the EOL Python2?

Reply:
@cclauss the last release of v2.7 is April. Let's drop 2.7 support after few more months

Reply:
> Should we also add a run on Python 3.8 just to make sure things work OK there?
 
Yes we can